### PR TITLE
Testing module

### PR DESCRIPTION
**Pull request by shella12**
### Testing To Do list: part 1

In this project, we wrote unit tests for the To Do list application. We wrote test cases for functions responsible for adding and deleting list items.  In order to complete this project we used the jest framework and following npm packages

-  jest 
-  jest-environment-jsdom 
- Babel

### CODE REVIEW

File: src/removeTask.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import todoArray from './__mocks__/todoarray';
+
+const removeTask = (list) => {
+  const index = Number(list.id);
+  list.remove();
+  todoArray.splice(index, 1);
+  todoArray.forEach((element, index) => {
+    element.index = index;
+  });
+  // localStorage.setItem('data', JSON.stringify(todoArray));
```
Comment (danmainah) [inline src/removeTask.js]: - [ ] @team kindly remove the commented code, it's not good practice to have commented code in your codebase.

File: src/add-remove.js [hunk 2]
```diff
@@ -90,6 +90,7 @@ export const addTask = (description, index, check) => {
   list.addEventListener('click', (event) => {
     editTask(event.target.id, list, checkboxImage, checkboxLabel);
   });
+  // console.log(list);
```
Comment (danmainah) [inline src/add-remove.js]: - [ ] @team kindly remove the commented code, it's not good practice to have commented code in your codebase.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @shella12 @mahdimahdawi and @megha-n-bodke, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Davidosky007 — APPROVED** [review]
Hi @shella12 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Good looking commit messages 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**